### DISCUSSION TITLE
Language statistics doesn't show files

### DISCUSSION THREAD

**Discussion by scripthunter7** [github-linguist/linguist#6077] [Q&A] [answered]
Hello!

If I click on "Adblock Filter List" in the statistics, the search page that appears doesn't show any files. What could be causing this? At first I thought it was some kind of cache problem, but the language has been live for 1+ week. 

Language type is data, and it was made visible in the statistics using the `linguist-detectable` attribute.

Example:
- Example repository: https://github.com/DandelionSprout/adfilt
- "Adblock Filter List" stats for this repo: https://github.com/DandelionSprout/adfilt/search?l=adblock-filter-list (but this doesn't work in other repositories either)

Screenshots:

<details>
<summary>Screenshot 1</summary>

<img src="https://user-images.githubusercontent.com/57285466/189721435-9df37008-13d5-4a7b-b28a-55069ada74fc.png" width="300px">
</details>

<details>
<summary>Screenshot 2</summary>

<img src="https://user-images.githubusercontent.com/57285466/189721275-c6283d83-458d-4c45-b7b4-f4472ef348ac.png" width="300px">
</details>

**Comment by lildude** [comment, marked as answer]
From https://github.com/github/linguist/issues/5952#issuecomment-1168505070

> You're looking at Search and its functionality which is completely independent of Linguist. It doesn't use Linguist. Instead it uses [go-enry](https://github.com/go-enry/go-enry) which pulls in the Linguist releases, however it tends to lag behind Linguist by a few weeks to months. It also requires our Search team to pull in the latest version as and when.
>
> So any time you see a language missing from Search and search results, it's likely because of this and the only option is to wait for go-enry to update and for this to then be pulled into Search.
